config/development.json

```diff
@@ -1,5 +1,5 @@
 {
-	"extends": "base",
+	"extends": "preview",
```

Comment:
This doesn't feel right. If we're trying to get the same beta product slugs in preview and development, we should think about moving the expanded list to `base` and limiting the list in `production.json`

Reply:
Sweet, matching `preview` and `dev` was definitely the intent. I'll update to limit the list in `production.json` 👍